camkes/runner/Context.py

```diff
@@ -485,11 +486,12 @@ def register_shared_variable(addr_space, obj_space, global_name, symbol, size, c
     size = int(size)
     frame_size = calc_frame_size(size, frame_size, obj_space.spec.arch)
     num_frames = size//frame_size
-
+    digits = str(int(log10(num_frames + 1)) + 1)
```

Comment:
Why is this using 'num_frames + 1'? It this to handle 0, but then have an additional '0' added at 9 already?